src/rlimitfloatenv.cpp

```diff
@@ -0,0 +1,82 @@
+#include <fenv.h>
+#include <sys/time.h>
+#include <sys/resource.h>
+#include "shareddata.h"
+#include "config.h"
+#include "dmtcp.h"
+
+/*************************************************************************
+ *
+ *  Save and restore rlimit and float exception settings.
+ *
+ *************************************************************************/
+
+namespace dmtcp
+{
+
+static int roundingMode = -1;
+static fenv_t envp;
+static rlim_t rlim_cur_nofile = 0;
+static rlim_t rlim_cur_stack = 0;
+
+static void
+save_rlimit_float_settings()
+{ roundingMode = fegetround();
```

Comment:
Newline after opening brace.

I wonder why wasn't it caught by the pre-commit hook. Perhaps the rules need to be updated to check for opening brace.